xbmc/cores/RetroPlayer/rendering/VideoRenderers/RPWinRenderer.cpp

```diff
@@ -69,7 +69,7 @@ bool CRPWinRenderer::Configure(AVPixelFormat format, unsigned int width, unsigne
   m_renderOrientation = orientation;
 
   m_targetPixFormat = AV_PIX_FMT_BGRA;
-  m_targetDxFormat = DXGI_FORMAT_B8G8R8X8_UNORM;
+  m_targetDxFormat = g_Windowing.GetBackBuffer()->GetFormat();
   if (g_Windowing.IsFormatSupport(DXGI_FORMAT_B8G8R8A8_UNORM, D3D11_USAGE_DYNAMIC))
```

Comment:
this (and line below) may be removed then

Reply:
Thanks